src/components/pages/dashboard/panels/map/azureMap.js

```diff
@@ -26,6 +26,11 @@ export class AzureMap extends Component {
     if (this.map) this.map.remove();
   }
 
+  shouldComponentUpdate(nextProps) {
+    // Component props never result in a dom updates from React
+    return false;
```

Comment:
If it will never update, even when/if the azureMapsKey changes, then why would the mapPanel say shouldComponentUpdate is true when the azureMapsKey changes?  If the azureMap won't respect the change, then its caller shouldn't care either, right?